CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+#### June 2024
+
+-----------
+**ConfigurationForm Changes**
+- We added options to provide default value and placeholder for the Field.
+- We Introduced  FieldSet  support, which allows to group the Fields and add visibility condition to control displaying on the form.
+- The FormField format is updated to support FieldSet. The single attribute should be used to assign the usual fields.
```

Comment:
NIT: can be changed from `usual fields` to `regular fields`.

Reply:
incorporated